### PR TITLE
[VIRTS-4151] Document split stdout/stderr

### PR DESCRIPTION
**Pull request by michael-the-jones**
## Description

Document recent change to include both stdout and stderr in result files. (Other changes to be made in Fieldmanual).

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Opened docs and API within CALDERA UI and verified documentation changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [N/A] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/api/v2/handlers/operation_api.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ async def get_operation_link(self, request: web.Request):
                           }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(LinkSchema(partial=True),
```
Comment (mchan143) [inline app/api/v2/handlers/operation_api.py:229]: Let's update this as discussed - creation of a `LinkResultSchema` in `app/api/v2/schemas` to capture the true response value of `dict(link=<Link display object>, result=<string>)`

File: app/api/v2/handlers/operation_api.py [hunk 2]
```diff
@@ -227,7 +227,7 @@ async def get_operation_link(self, request: web.Request):
...
-                                     description='Contains a result string for the link requested.')
+                                     description='Contains a result dictionary as a string for the link requested.')
```
Comment (mchan143) [inline app/api/v2/handlers/operation_api.py:230]:
```suggestion
                                     description='Contains a dictionary of the requested link object and link's result.')
```

Comment (mchan143) [reply app/api/v2/handlers/operation_api.py:230]: Added a little clarification to this one, but feel free to tweak as you see fit. You could also add in the note about the base64 encoding if you feel it's appropriate.

File: app/api/v2/handlers/operation_api.py [hunk 3]
```diff
@@ -226,8 +227,8 @@ async def get_operation_link(self, request: web.Request):
                               'description': 'UUID of the link object to retrieve results of.'
                           }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(LinkSchema(partial=True),
-                                     description='Contains a result string for the link requested.')
+    @aiohttp_apispec.response_schema(LinkResultSchema(partial=True),
```
Comment (mchan143) [inline app/api/v2/handlers/operation_api.py:230]: Is there a case where we'd want `partial=True`?

```suggestion
    @aiohttp_apispec.response_schema(LinkResultSchema(),
```

Comment (michael-the-jones) [reply app/api/v2/handlers/operation_api.py]: I moved this to the nested LinkSchema inside of LinkResultSchema

**Review by mchan143 — APPROVED** [review]
LGTM

**Review by kaylakraines — APPROVED** [review]
Looks good!

**Comment by mchan143** [comment]
@elegantmoose @clenk can we get your review to merge this?

**Review by elegantmoose — APPROVED** [review]

**Pull request merged**